### ISSUE TITLE
MCOParameters: "MCO" is excessive

### ISSUE THREAD

**Issue by Corwinpro** [force-h2020/force-bdss#294]
It might be worth rethinking the naming of the `BaseMCOParameter` and its subclasses. The word `MCO` sounds excessive in this case, as we don't have any parameters other than MCO parameters. Moreover, the `MCOParameter` class was explicitly designed to define the parameters, related to the MCO only.

By removing the `MCO` from the names, we would
- simplify the interface of this class, as it would be less restrictive
- shorten the names of the associated classes, which makes life a bit easier for the developers

**Comment by flongford** [comment]
Agreed, we should also look at creating a module for the subclasses of `BaseMCOParameter` that we provide in the `force_bdss`. 

Ideally they should be contributed in a `CorePlugin` object, with a stand alone `BaseMCOFactory` subclass
